app/views/pet_applications/show.html.erb

```diff
@@ -12,10 +12,12 @@
 <div><h3>Pet Applications</h3>
   <%= form_tag "/pet_applications/#{@pet_application.id}/approve_application", method: :patch do %>
     <% @pet_application.pets.each do |pet| %>
-      <div id="app-<%= @pet_application.id %>-pet-<%= pet.id %>">
-        <%= check_box_tag("pet_id[]", "#{pet.id}") %>
-        <%= label_tag("pet_id[]", "#{pet.name}") %><br/>
-      </div>
+      <% unless pet.adopt_status == "Pending" %>
```

Comment:
This is one of the lines that is causing one of your errors.  I've got it fixed in my us18 edgecase fix